src/coprocessor/statistics/analyze.rs

```diff
@@ -44,6 +45,15 @@ use crate::{
 const ANALYZE_VERSION_V1: i32 = 1;
 const ANALYZE_VERSION_V2: i32 = 2;
 
+// minimum number of core kept for analyze
+const ANALYZER_CORE_LOWER_BOUND: usize = 1;
+// max number of cores kept for analyze = core_number - ANALYZER_CORE_EXCLUSIVE_BOUND
+const ANALYZER_CORE_EXCLUSIVE_BOUND: usize = 1;
```

Comment:
1 is too aggressive for 8vCPU TiKV, the CPU usage of TiKV is hard to reach 7vCPU since there is IO waits.